content/docs/server_setup/server_software.mdx

```diff
@@ -0,0 +1,222 @@
+
+<Callout type="warn" title="Warning">
+  While Paper features many optimizations and features, many of these affect vanilla gameplay mechanics.\
+  This makes it unsuitable for servers that wish to maintain a vanilla experience or those who make complex farms that utilize intricate vanilla mechanics.
+</Callout>
+
+<Card href="https://jar.setup.md/download/paper/latest/latest" title="Download">
+  Download a copy of the latest PaperMC server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Pufferfish (recommended)
+
+Pufferfish is a [fork](#backend-servers) of [Paper](#paper) that adds significant optimizations to the game without much impact to normal gameplay.\
+These optimizations include entity, hopper, SIMD and other performance improvements.\
+This is highly recommended for all servers due to the significant performance improvements it brings, without affecting plugin compatibility.
+
+<Card href="https://pufferfish.host/downloads" title="Download">
+  Download a copy of the latest Pufferfish server software via pufferfish.host.\
+  **Do not attempt to use Pufferfish+ or Pufferfish+ (Purpur). This is a paid
+  server software and is mostly used only by the customers of Pufferfish Host
+  itself**.
+</Card>
+
+## Purpur
+
+[Purpur](https://purpurmc.org/) is another fork of Paper that adds additional configuration concerning different mechanics of the game, removing the need to download or create additional plugins.\
+It also features some performance patches from [Pufferfish](#pufferfish-recommended).
+It is recommended to use Pufferfish over Purpur, unless you require the additional features and configuration that Purpur provides.
+
+<Card
+  href="https://jar.setup.md/download/purpur/latest/latest"
+  title="Download">
+  Download a copy of the latest Purpur server software via
+  [jar.setup.md](https://jar.setup.md/).
+</Card>
+
+## Folia
+
+[Folia](https://papermc.io/software/folia) is a fork of Paper that splits the worlds up into "regions" which all run relatively independently.\
+Each region features its own tick loop (ticking at Minecraft's regular 20TPS), running on a thread pool in parallel.\
+This allows for better performance on servers with many players and entities, who are spread far out.
+
+Folia was created because Minecraft is single-threaded, meaning it cannot take advantage of all of your CPU, and instead can only use a small portion of it.\
+Softwares such as Paper already help to alleviate this issue, but Folia takes it a step further by splitting the world up into regions.
+
+<Callout type="warn" title="Warning">
+    **Folia is not completely usable!**\
+    You must consider whether you need to use it. Folia is not completely stable yet, although it is stable enough to be used in production, one famous example being 2b2t.\
```

Comment:
Does 2b2t use Folia? If so, I would appreciate a source being linked here.

Reply:
There's actually a few videos with proof (confirmed folia bug reports visible on the 2b2t server) and some have said 2b2t use Folia in AC - https://discord.com/channels/348681414260293634/1267660978451906673/1267671746635370547
https://discord.com/channels/348681414260293634/348681414260293635/1144940853496922123
https://discord.com/channels/348681414260293634/1247565563853668353/1247573158597365790
https://discord.com/channels/348681414260293634/348681414260293635/1197460852106866750


Unsure which one you would like to be linked, or if you want me to remove the reference to 2B2T completely.